WhiteDentalClinic/WhiteDentalClinic.Application/Models/MedicalServiceModel/ResponseMedicalServices.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WhiteDentalClinic.Application.Models.MedicalServiceModel
+{
+    public class ResponseMedicalServices
```

Comment:
All the other classes are called "Something**Response**" only this one is "**Response**Something". I think the second option is way better and should be used everywhere as a pattern.